src/ecotranslations.jl

```diff
@@ -0,0 +1,13 @@
+# Getters and setters with microbiome-flavored names
+
+# species -> features
+nfeatures(com::AbstractComMatrix) = nspecies(com)
+getfeature(com::AbstractComMatrix, idx) = getspecies(com, idx)
+featurenames(com::AbstractComMatrix) = specnames(com)
+featuretotals(com::AbstractComMatrix) = speciestotals(com)
```

Comment:
It is slightly uncertain what speciestotals and sitetotals mean (they are total abundances when the matrix has eltype Int, but there are other possibilities, so I think they might need to be renamed in SpatialEcology

Reply:
If the eltype isn't Int? I'd still think of it as a total, even if it ends up just being 1.0 for a relative abundance table.